### PR TITLE
Add information about DevServices in Redis documentation

### PR DESCRIPTION
**Pull request by netodevel**
Adds properties available to customize the redis testcontainer.

### CODE REVIEW

File: docs/src/main/asciidoc/redis.adoc [hunk 1]
```diff
@@ -592,6 +592,14 @@ Quarkus supports a feature called DevServices that allows you to create various
 What that means practically, is that if you have docker running and have not configured `quarkus.redis.hosts`,
 Quarkus will automatically start a Redis container when running tests or dev-mode, and automatically configure the connection.
 
+Available properties to customize the Redis DevServices.
+[source,properties]
+----
+quarkus.redis.devservices.image-name=redis:6.0-alpine
+quarkus.redis.devservices.port=6377
+quarkus.redis.devservices.enabled=true (true by default)
```
Comment (machi1990) [inline docs/src/main/asciidoc/redis.adoc:600]:
```suggestion
quarkus.redis.devservices.enabled=true
```

**Review by machi1990 — CHANGES_REQUESTED** [review]
Thanks for this addition @NetoDevel I added a small suggestion.

File: docs/src/main/asciidoc/redis.adoc (hunk 1 above)
Comment (machi1990) [reply docs/src/main/asciidoc/redis.adoc:600]: No need for this as we have the generated doc which tells us what default values are

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks, I added a small suggestion too.

File: docs/src/main/asciidoc/redis.adoc [hunk 2]
```diff
@@ -592,6 +592,14 @@ Quarkus supports a feature called DevServices that allows you to create various
 What that means practically, is that if you have docker running and have not configured `quarkus.redis.hosts`,
 Quarkus will automatically start a Redis container when running tests or dev-mode, and automatically configure the connection.
 
+Available properties to customize the Redis DevServices.
```
[the remark this answers is not in the document]
Comment (NetoDevel) [reply docs/src/main/asciidoc/redis.adoc]: @gsmet, done

**Review by machi1990 — DISMISSED** [review]
LGTM. Can you squash then it should be good to merge?

**Review by gsmet — APPROVED** [review]

**Review by gastaldi — CHANGES_REQUESTED** [review]
Include the generated asciidoc instead:

```asciidoc
include::{generated-dir}/config/quarkus-redis-client-config-group-dev-services-config.adoc[opts=optional, leveloffset=+1]
```

**Comment by gastaldi** [comment]
> I need to create a separate specific file containing only redis-devservices configs
> and then do an include in that redis.adoc?

No, this file is generated automatically when you build the project.

**Comment by NetoDevel** [comment]
@gastaldi, the first time I did squash, please check that I did it correctly :smile:

**Comment by gastaldi** [comment]
Your commit message is weird. You can fix this by doing `git commit --amend`

**Comment by NetoDevel** [comment]
@gastaldi, feito

**Review by gastaldi — APPROVED** [review]
Fantastic job!

**Pull request merged**